Can't install e2e apk on Android 12 and higher

# Bug Report


## Problem 
To keep our e2e tests up to date we need to use relevant Android version. SauceLabs emulators are x86_64 only. And now it's not possible to install and use e2e application on such emulators with Android 12 or more.


#### Expected behavior
The e2e apk is installed on Android 12

#### Actual behavior
<img width="384" alt="Screenshot 2023-09-12 at 17 21 13" src="https://github.com/status-im/status-mobile/assets/29711298/2e146d01-f97b-46b4-9722-b380daaad2c2">


### Reproduction

1. Create an Android emulator with x86 or x86_64 ABI and API level 31 or more
2. Try installing e2e application 

### Additional Information

- Status version: nightly e2e
- Operating System: Android 12 and higher 


[comment]: # (Please, add logs/notes if necessary)

This issue at android ndk repo caught my attention : https://github.com/android/ndk/issues/1298 

its a seccomp issue with x86_64 emulators which was fixed in ndk r26 and probably backported to ndk r25(not sure)
However the error message is not exactly the same as what we face here but I figured its worth a shot to try upgrading ndk to this version to see if it fixes our crash.

we didn't have ndkversion 26 in our nixpkgs so I added them to our fork here : https://github.com/status-im/nixpkgs/commit/cf1c475a5141631ff85d48fbb5e015945df50d37

Unfortunately I couldn't get status-go to build on my mac and it would fail with this error : 

```
 > Building status-go for: android/arm,android/arm64,android/386
 > /nix/store/m1iynw68yqby0xi18djqfqvqh6z52528-gomobile-unstable-2022-05-18/bin/gomobile: 
 go build -tags gowaku_skip_migrations,gowaku_no_rln -ldflags -X github.com/status-im/status-
 go/params.GitCommit=27b770c41bd2896ebc30b01985eeea1fbe642fba -X github.com/status-im/status-
 go/params.IpfsGatewayURL=https://ipfs.status.im/ -X github.com/status-im/status-go/params.Version=0.171.6 
 -s -w -buildmode=c-shared -o=/private/tmp/nix-build-status-go-0.171.6-27b770c-android.drv-0/gomobile-
 work/android/src/main/jniLibs/arm64-v8a/libgojni.so ./gobind failed: exit status 2
 > # github.com/status-im/status-go/vendor/github.com/mutecomm/go-sqlcipher/v4
 > aesce.c:194:18: error: always_inline function 'vaesimcq_u8' requires target feature 'aes',
  but would be inlined into
  function 'mbedtls_aesce_inverse_key' that is compiled without support for 'aes'
make: *** [build-android] Error 1
```

probably @caybro or @alaibe might have an idea on how to get this to work.

I traced down the error to https://github.com/status-im/go-sqlcipher/blob/master/aesce.c#L194 in our fork of `go-sqlcipher `package but I wasn't sure If its a build flag issue or something else.

I also tried `ndkVersion` `25.2.9519653 `
I do get a successful build, but on x86_64 simulators the app crashes right after generating keys stage which means that the issue is still present :)

--- a/android/gradle.properties
+++ b/android/gradle.properties
@@ -42,7 +42,7 @@ KEYSTORE_KEY_PASSWORD=password
 # By default we build a mostly universal APK
 ANDROID_ABI_SPLIT=false
 # Some platforms are excluded though
-ANDROID_ABI_INCLUDE=armeabi-v7a;arm64-v8a;x86
+ANDROID_ABI_INCLUDE=armeabi-v7a;arm64-v8a;x86;x86_64
 
 org.gradle.jvmargs=-Xmx8704M
 

--- a/ci/Jenkinsfile.android
+++ b/ci/Jenkinsfile.android
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-library 'status-jenkins-lib@v1.7.15'
+library 'status-jenkins-lib@switch_e2e_to_x86_64'
 
 /* Options section can't access functions in objects. */
 def isPRBuild = utils.isPRBuild()

--- a/ci/Jenkinsfile.android
+++ b/ci/Jenkinsfile.android
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-library 'status-jenkins-lib@switch_e2e_to_x86_64'
+library 'status-jenkins-lib@v1.8.4'
 
 /* Options section can't access functions in objects. */
 def isPRBuild = utils.isPRBuild()

--- a/ci/Jenkinsfile.combined
+++ b/ci/Jenkinsfile.combined
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-library 'status-jenkins-lib@v1.7.15'
+library 'status-jenkins-lib@v1.8.4'
 
 pipeline {
   agent { label 'linux' }

--- a/ci/Jenkinsfile.e2e-nightly
+++ b/ci/Jenkinsfile.e2e-nightly
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-library 'status-jenkins-lib@v1.7.15'
+library 'status-jenkins-lib@v1.8.4'
 
 pipeline {
   agent { label 'linux' }

--- a/ci/Jenkinsfile.ios
+++ b/ci/Jenkinsfile.ios
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-library 'status-jenkins-lib@v1.7.15'
+library 'status-jenkins-lib@v1.8.4'
 
 /* Options section can't access functions in objects. */
 def isPRBuild = utils.isPRBuild()

--- a/ci/Jenkinsfile.tests
+++ b/ci/Jenkinsfile.tests
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-library 'status-jenkins-lib@v1.7.15'
+library 'status-jenkins-lib@v1.8.4'
 
 /* Options section can't access functions in objects. */
 def isPRBuild = utils.isPRBuild()

